model/jpa/src/main/java/org/keycloak/authorization/jpa/entities/ResourceServerEntity.java

```diff
@@ -66,6 +70,17 @@ public void setPolicyEnforcementMode(PolicyEnforcementMode policyEnforcementMode
         this.policyEnforcementMode = policyEnforcementMode;
     }
 
+    public void setDecisionStrategy(DecisionStrategy decisionStrategy) {
```

Comment:
@pedroigor based on the change below. Would make sense to remove CONSENSUS from https://github.com/keycloak/keycloak-documentation/blob/532a46f60f04ef709776d1599359bc9ab24e10a4/authorization_services/topics/policy-aggregated-policy.adoc#decision-strategy-for-aggregated-policies  and https://github.com/keycloak/keycloak-documentation/blob/532a46f60f04ef709776d1599359bc9ab24e10a4/authorization_services/topics/permission-decision-strategy.adoc#policy-decision-strategies ?